src/navigation/_ComponentToolbar.jsx

```diff
@@ -61,17 +63,17 @@ const ComponentToolbar = ({
         {hasThemes && (
           <MenuButton
             text="Theme"
-            items={devToolsConfig.themes}
-            selectedKey={devToolsConfig.selectedTheme}
-            onChange={devToolsConfig.onChangeTheme}
+            items={Object.keys(appSettings.themes)}
+            selectedKey={appSettings.state.theme}
```

Comment:
I think it would be easier to reason with the 'state' vs settings keys if the state keys were flattened to `appSettings.locale`, `appSettings.theme`, etc

Reply:
i flattened this to currrentLocal etc. I thought that was more fitting since we also have the default values in the same appSettings object.